book/chapters/chapter3/evaluation_and_benchmarking.qmd

```diff
@@ -285,6 +285,31 @@ print(plt2)
 ```
 
 
+### Confidence Intervals {#sec-resampling-ci}
+
+Confidence intervals (CIs) provide a range of values within which we can be confident that it covers the true generalization error.
+Instead of relying solely on a single point estimate, CIs offer a measure of uncertainty around this estimate, allowing us to understand the reliability of our performance estimate.
+While constructing CIs for the generalization error is challenging due to the complex nature of the inference problem, some methods have been shown to work well in practice @kuempelfischer2024ciforge.
```

Comment:
I would add some more context here -- some learners/models can provide these directly (and often those calculations aren't all that complex), but if the learner doesn't support it, we have to do something else. Then describe in a sentence or two what those methods do.